helm/trino-gateway/values.yaml

```diff
@@ -110,6 +111,11 @@ tolerations: []
 
 affinity: {}
 
+commonLabels: {}
+  # io.example/cost-center: "000000"
+  # io.example/team: "trino"
```

Comment:
Do the labels have to have a namespace or can it also be

example: "some value"

Reply:
If yes .. we should add such a simpler example. Also

Reply:
It does need to have namespace. That was just the way we were using commonLabels. I can change them and add comment for docs.